src/test/scala/io/iohk/ethereum/BlockHelpers.scala

```diff
@@ -0,0 +1,25 @@
+package io.iohk.ethereum
+
+import akka.util.ByteString
+import io.iohk.ethereum.domain.Block
+
+trait BlockHelpers {
```

Comment:
Could you make it an object? I'd move helpers I extracted there: https://github.com/input-output-hk/mantis/pull/735/files#diff-840c293a5c8d271766b7ad38d72f23fdc2a5d0406db32ee60fd727cc6615ca75R1 into the same package, so less conflicts and/or code duplicates are created